app/Http/Controllers/BaseController.php

```diff
@@ -21,6 +21,7 @@ class BaseController extends \Illuminate\Routing\Controller
     }
     use DispatchesJobs, RedirectsUsers, ValidatesRequests;
 
+    /** @var Account */
     protected $account;
```

Comment:
Nitpick: 
```suggestion
    protected Account $account;
```

Reply:
Crashes elsewhere due to access before initialisation annoyingly, will dig in to that in a separate or potentially